City-of-Helsinki/helsinki-paatokset
# PP-232: Handle meeting documents and agenda files

This adds handling for meeting documents and agenda files as media entities in Drupal.

**To test**
- Checkout branch
- Run `make new` or SSH into the container and run `drush cr; drush cim -y`
- You might have to reset the Helsinki Kanava settings after config import: https://helsinki-paatokset.docker.so/fi/admin/config/helsinki-kanava/settings
- Inside the container, run: `drush ap:fs; drush mim ahjo_meetings:latest`
- Check that Ahjo Document media items were created:  https://helsinki-paatokset.docker.so/fi/admin/content/media?name=&type=ahjo_document&status=All&langcode=All
  - Check a few and make sure fields were actually populated based on the data stored in the JSON field. The Issued field might be empty on most files.
- Check that the meetings listing has content: https://helsinki-paatokset.docker.so/fi/admin/content/meetings
  - Find a meeting with both Agenda and Minutes published and go into the edit view
  - Make sure the meeting has media references in the **Meeting Data > Agenda** and **Meeting Data > Meeting Documents** sections
